opensrp-app/src/main/java/org/smartregister/view/activity/SecuredNativeSmartRegisterActivity.java

```diff
@@ -1,647 +1,654 @@
-package org.smartregister.view.activity;
```

Comment:
Not sure why we have this large diff here?

Reply:
https://github.com/OpenSRP/opensrp-client-core/pull/715#discussion_r551815077

Reply:
https://github.com/OpenSRP/opensrp-client-core/pull/715#discussion_r551863135